assets/scss/modules/_buttons.scss

```diff
@@ -89,6 +89,30 @@
 
     }
 
+    &.secondary- {
+      &green {
+        background: transparent;
+        border: 1px solid $dp-color-green;
+        color: $dp-color-green;
+
+        &:hover {
+          background: $dp-color-green-brightness;
+        }
+      }
+
+      &brown {
+        background: transparent;
+        border: 1px solid $dp-color-brown;
+        color: $dp-color-brown;
+
+        &:hover {
+          background: $dp-color-brown-brightness;
+          color: $dp-color-brown;
```

Comment:
maybe don't need this. it's the font color define in the css class

Reply:
It's true, thanks @fcoronelmakingsense